src/game.py

```diff
@@ -1,22 +1,51 @@
+from src.hand import *
+
 class Game:
     # init game with deck, discard deck, and player count
-    def __init__(self, player_count)
+    def __init__(self, player_count):
+        game_deck = Deck(True)
+        discard_deck = Deck()
+
+        hands = []
+
+        for i in range(0, player_count):
+            hands.append(Hand(game_deck))
 
     # resets the deck
     def reset(self):
+        print("I compile")
```

Comment:
does reset reset everything, or just one hand or renew just the deck?

Reply:
You made this method. I took it as a helper method that resets the game and discard decks when you start a new round.

Reply:
ok delete it

Reply:
your implementation is fine